srcpkgs/wineasio/template

```diff
@@ -1,30 +1,32 @@
 # Template file for 'wineasio'
 pkgname=wineasio
-version=0.9.2
-revision=7
-archs="i686"
-repository="nonfree"
+version=1.0.0
+revision=1
+archs="i686 x86_64"
 build_style=gnu-makefile
-create_wrksrc=yes
-build_wrksrc=${pkgname}
-homepage="https://sourceforge.net/projects/wineasio"
-hostmakedepends="unzip wine-tools"
+_rtaudio_ver=5.1.0
+hostmakedepends="unzip wine-tools pkg-config"
 makedepends="wine wine-devel jack-devel"
 depends="jack"
 short_desc="JACK driver for WINE to provide Windows Apps with ASIO support"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="LGPL-2.1, Steinberg"
-distfiles="${SOURCEFORGE_SITE}/wineasio/${pkgname}-${version}.tar.gz
-	https://www.steinberg.net/sdk_downloads/asiosdk2.3.zip"
-checksum="9fbc2d10a4cec307dc0558bed82b887c864424d16da40a80f1d80f907d92af0b
-	69225020e9144af568784c6076b55939a55369edaf0ffcdf4d5e5b5898f31b9b"
+license="GPL-2.0-or-later,  LGPL-2.1-or-later"
+homepage="https://sourceforge.net/projects/wineasio"
+distfiles="https://github.com/wineasio/wineasio/archive/refs/tags/v${version}.tar.gz
+	https://github.com/falkTX/rtaudio/archive/refs/tags/${_rtaudio_ver}.tar.gz"
+checksum="6938457947373c3c3411c69a9cd3a6835191a04ec694d477b0ecf59161a98d28
+ 42242eb0db5902166a38530c242567294510f57be22754cfafe31ed1f35b76bf"
+
+case "$XBPS_TARGET_MACHINE" in
+	i686) make_build_target=32 ;;
+	x86_64) make_build_target=64 ;;
+esac
 
-pre_configure() {
-	cp -v ../ASIOSDK2.3/common/asio.h asio.h
+post_extract() {
+	cp -ar "${XBPS_BUILDDIR}/rtaudio-${_rtaudio_ver}"/* "${wrksrc}/rtaudio"
```

Comment:
`mv  ../rt... rtaudio`, and if necessary `rmdir rtaudio` first.

Reply:
Thanks. Slavish devotion to however it was done in the previous template makes for some uglier things sometimes.